creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
+[Quasi-connectivity](https://minecraft.fandom.com/wiki/Tutorials/Quasi-connectivity), a bug-turned-feature in Java Edition, is not present in Bedrock Edition. Systems that utilize mechanics such as "Block Update Detector" (BUD) switches won't work. Pistons also require one tick to retract, and won't leave blocks behind if given a one-tick pulse.
 
-**Resource Packs**
+While Java has a deterministic albeit directional order for redstone updates, Bedrock's update order is random, meaning a circuit that tries to have multiple blocks do something at the same moment may behave differently every time it's activated.
 
-There are a lot of similarities between Bedrock Edition and Java Edition when it comes to resource packs but they do contain some differences. The obvious one is the swap of `.mcmeta` files with `.json` files in Bedrock Edition. These files are used to define the properties of different parts of the interface and are largely used in the same way with distinct syntax differences between the two formats. You’ll also see large differences in texture file formats, the main being the use of TGA files for work with alpha channels instead of PNG. Some textures (mainly entities) are laid out slightly different as well.
+## Resource Packs
+The idea behind resource packs is the same in both editions: change how the game looks. However, the capabilities and layout are very different.
 
-**Behavior Packs**
+Only Java Edition can change the shape of blocks, and only Bedrock Edition can change the shape of entities. Each uses a geometry format for this purpose that's incompatible with the other edition.
 
-One of the biggest differences between Bedrock Edition and Java Edition is the use of behavior packs. While functionally similar to the data packs in Java Edition, the actual implementation and use of behavior packs vary quite a bit.
+Many image files are named differently, since Java updated most of its names in 1.13 to be more consistent and Bedrock did not.
 
-Behavior packs bring in new functionality whether it be building your own entities from scratch, adding new blocks and items, or access to events using the JavaScript API. It allows for an enormous amount of flexibility and control and is one of the most powerful parts of Bedrock Edition when compared against Java Edition.
+To animate textures, Bedrock uses a single file called `flipbook_textures.json`, while Java uses individual `.mcmeta` files for each texture.
 
-**Gameplay and Player Input**
+Java Edition can create custom fonts and GLSL shaders, while Bedrock cannot. Bedrock Edition can create custom particles and fogs, while Java cannot.
 
```

Comment:
First, differences in related concepts: remodeling stuff and retexturing stuff and animating stuff
Next, version-exclusive functionality